parsers/powershell.c

```diff
@@ -456,7 +479,7 @@ static bool parseFunction (tokenInfo *const token)
 				case TOKEN_STRING:			vStringCatS (arglist, "'...'");	break;
 
 				case TOKEN_IDENTIFIER:
-				case TOKEN_KEYWORD:
+				case TOKEN_KEYWORD_OTHER:
```

Comment:
Where is TOKEN_KEYWORD_OTHER assigned to `->type`.

Reply:
In [another comment](https://github.com/universal-ctags/ctags/pull/3476#issuecomment-1236221400), masatake suggested leaving `TOKEN_KEYWORD` as it is and adding an attribute like `token->keywordType`.

That proposal shall be addressed in a separate PR in accordance with [this comment](https://github.com/universal-ctags/ctags/pull/3476#issuecomment-1236241369).

In this PR, remove the unused TOKEN_KEYWORD_OTHER.